bridge/client/app/_components/ktb-stage-details/ktb-stage-details.component.html

```diff
@@ -74,6 +74,7 @@ <h2 class="m-0" [textContent]="service.serviceName"></h2>
                     <ktb-evaluation-details [evaluationData]="evaluation.getFinishedEvent()" [showChart]="false"></ktb-evaluation-details>
                   </ng-template>
                 </dt-tag-list>
+                <span> - go to: <a [routerLink]="['/project', failedRootEvent.getProject(), 'service', service.serviceName, 'context', failedRootEvent.shkeptncontext, 'stage', selectedStage.stageName]">Evaluation result</a></span>
```

Comment:
I would just provide the Link to "Evaluation result". Without the "go to:", which seems unnecessary to me.